packages/engine/src/getContext.js

```diff
@@ -57,33 +57,32 @@ const blockData = ({
   visible,
 });
 
-const getContext = async ({ block, contextId, pageId, rootContext }) => {
-  if (rootContext.contexts[contextId]) {
-    rootContext.contexts[contextId].update();
-    return rootContext.contexts[contextId];
+const getContext = async ({ block, contextId, lowdefy }) => {
+  if (lowdefy.contexts[contextId]) {
+    lowdefy.contexts[contextId].update();
+    return lowdefy.contexts[contextId];
   }
   if (!block) {
     throw new Error('A block must be provided to get context.');
   }
   // eslint-disable-next-line no-param-reassign
-  if (!rootContext.input[contextId]) {
-    rootContext.input[contextId] = {};
+  if (!lowdefy.inputs[contextId]) {
+    lowdefy.inputs[contextId] = {};
   }
-  rootContext.contexts[contextId] = {
+  lowdefy.contexts[contextId] = {
     id: contextId,
     blockId: block.blockId,
     eventLog: [],
-    pageId,
     requests: {},
-    root: rootContext,
+    root: lowdefy,
```

Comment:
Maybe we can rename root here to lowdefy as well?

Reply:
rename all the things 🐙